### PR TITLE
[zh-cn] Sync 1.25/configure-liveness-readiness-startup-probes.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Sync the upstream:
```yaml
content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md
```

### CODE REVIEW

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -785,43 +790,50 @@ was set.
...
-pod- and probe-level `terminationGracePeriodSeconds` are set, the kubelet will
-use the probe-level value.
+In 1.25 and beyond, users can specify a probe-level `terminationGracePeriodSeconds`
+as part of the probe specification. When both a pod- and probe-level 
+`terminationGracePeriodSeconds` are set, the kubelet will use the probe-level value.
 -->
-在 1.21 及更高版本中，当特性门控 `ProbeTerminationGracePeriod` 被启用时，
-用户可以指定一个探测器层面的 `terminationGracePeriodSeconds` 作为探测器规约的一部分。
-当该特性门控被启用，并且 Pod 层面和探测器层面的 `terminationGracePeriodSeconds`
+在 1.21 及更高版本中，用户可以指定一个探测器层面的 `terminationGracePeriodSeconds`
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:797]: probe 一般翻译为“探针”

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 2]
```diff
@@ -785,43 +790,50 @@ was set.
...
+by default. For users choosing to disable this feature, please note the following:
+
+* The `ProbeTerminationGracePeriod` feature gate is only available on the API Server. 
+The kubelet always honors the probe-level `terminationGracePeriodSeconds` field if 
+it is present on a Pod.
+
 -->
-在 Kubernetes 1.22 中，`ProbeTerminationGracePeriod` 特性门控只能用在 API 服务器上。
+{{< note >}}
+从 Kubernetes 1.25 开始，默认启用 `ProbeTerminationGracePeriod` 功能。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:812]:
```suggestion
从 Kubernetes 1.25 开始，默认启用 `ProbeTerminationGracePeriod` 特性。
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 3]
```diff
@@ -785,43 +790,50 @@ was set.
...
+选择禁用此功能的用户，请注意以下事项:
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:813]:
```suggestion
选择禁用此特性的用户，请注意以下事项:
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 4]
```diff
@@ -785,43 +790,50 @@ was set.
...
+
+* `ProbeTerminationGracePeriod` 特性门控只能用在 API 服务器上。
 kubelet 始终遵守探针级别 `terminationGracePeriodSeconds` 字段
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:816]:
```suggestion
  kubelet 始终遵守探针级别 `terminationGracePeriodSeconds` 字段
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 5]
```diff
@@ -785,43 +790,50 @@ was set.
...
 （如果它存在于 Pod 上）。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:817]:
```suggestion
  （如果它存在于 Pod 上）。
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 6]
```diff
@@ -785,43 +790,50 @@ was set.
...
 
 <!--
-If you have existing Pods where the `terminationGracePeriodSeconds` field is set and
+* If you have existing Pods where the `terminationGracePeriodSeconds` field is set and
 you no longer wish to use per-probe termination grace periods, you must delete
 those existing Pods.
 -->
-如果你已经为现有 Pod 设置了 `terminationGracePeriodSeconds`
+* 如果你已经为现有 Pod 设置了 `terminationGracePeriodSeconds`
 字段并且不再希望使用针对每个探针的终止宽限期，则必须删除现有的这类 Pod。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:825]:
```suggestion
  字段并且不再希望使用针对每个探针的终止宽限期，则必须删除现有的这类 Pod。
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 7]
```diff
@@ -785,43 +790,50 @@ was set.
...
-When you (or the control plane, or some other component) create replacement
+* When you (or the control plane, or some other component) create replacement
 Pods, and the feature gate `ProbeTerminationGracePeriod` is disabled, then the
 API server ignores the Probe-level `terminationGracePeriodSeconds` field, even if
 a Pod or pod template specifies it.
 -->
-当你（或控制平面或某些其他组件）创建替换 Pod，并且特性门控 `ProbeTerminationGracePeriod`
+* 当你（或控制平面或某些其他组件）创建替换 Pod，并且特性门控 `ProbeTerminationGracePeriod`
 被禁用时，API 服务器会忽略探针级别的 `terminationGracePeriodSeconds` 字段设置，
 即使 Pod 或 Pod 模板指定了它。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:835]:
```suggestion
  被禁用时，API 服务器会忽略探针级别的 `terminationGracePeriodSeconds` 字段设置，
  即使 Pod 或 Pod 模板指定了它。
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 8]
```diff
@@ -785,43 +790,50 @@ was set.
...
 -->
-在 Kubernetes 1.22 中，`ProbeTerminationGracePeriod` 特性门控只能用在 API 服务器上。
-kubelet 始终遵守探针级别 `terminationGracePeriodSeconds` 字段
-（如果它存在于 Pod 上）。
+{{< note >}}
+从 Kubernetes 1.25 开始，默认启用 `ProbeTerminationGracePeriod` 特性。
+选择禁用此特性的用户，请注意以下事项:
+
+* `ProbeTerminationGracePeriod` 特性门控只能用在 API 服务器上。
+  kubelet 始终遵守探针级别 `terminationGracePeriodSeconds` 字段
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:816]:
```suggestion
  kubelet 始终优先使用探针级别 `terminationGracePeriodSeconds` 字段
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 9]
```diff
@@ -785,43 +790,50 @@ was set.
...
 Pods, and the feature gate `ProbeTerminationGracePeriod` is disabled, then the
 API server ignores the Probe-level `terminationGracePeriodSeconds` field, even if
 a Pod or pod template specifies it.
 -->
-当你（或控制平面或某些其他组件）创建替换 Pod，并且特性门控 `ProbeTerminationGracePeriod`
-被禁用时，API 服务器会忽略探针级别的 `terminationGracePeriodSeconds` 字段设置，
-即使 Pod 或 Pod 模板指定了它。
+* 当你（或控制平面或某些其他组件）创建替换 Pod，并且特性门控 `ProbeTerminationGracePeriod`
+  被禁用时，API 服务器会忽略探针级别的 `terminationGracePeriodSeconds` 字段设置，
+  即使 Pod 或 Pod 模板指定了它。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:835]:
```suggestion
  被禁用时，即使 Pod 或 Pod 模板指定了 `terminationGracePeriodSeconds` 字段，
  API 服务器也会忽略探针级别的 `terminationGracePeriodSeconds` 字段设置。
```

**Review by Sea-n — COMMENTED** [review]
/approve

**Pull request merged**